### PR TITLE
Feat/modem db 2.0.1

### PR DESCRIPTION
**Pull request by kylebonnici**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 2.4.1 - 2024-07-09
```
Comment (KievDevel) [inline Changelog.md:1]: unreleased I assume
```suggestion
## 2.4.3 - unreleased
```

**Comment by KievDevel** [comment]
Does it overlap with https://github.com/NordicSemiconductor/pc-nrfconnect-cellularmonitor/releases/tag/v2.4.1?

File: Changelog.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
 ## 2.4.3 - unreleased
 
+### Added
+
+-   Pre-bundled Modem Trace Databases v1.3.6, v1.3.6-FOTA, v1.3.7, v1.3.7-FOTA,
+    v2.0.1, v2.0.1-FOTA, v2.0.2 and v2.0.2-FOTA.
```
Comment (greg-fer) [inline Changelog.md:6]:
```suggestion
-   Pre-bundled Modem Trace Databases: v1.3.6, v1.3.6-FOTA, v1.3.7, v1.3.7-FOTA,
    v2.0.1, v2.0.1-FOTA, v2.0.2, and v2.0.2-FOTA.
```

File: Changelog.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
+
 ### Fixed
 
+-   Modem Trace Database dropdown filters out unsupported DB for nRF9151.
```
Comment (greg-fer) [inline Changelog.md:10]:
```suggestion
-   Issue where the Modem Trace Database's drop-down would filter out an unsupported database for nRF9151.
```
How is filtering out an _unsupported_ db an issue? Sounds logical to me to filter out something that is not supported.

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**